plugin/pkg/managedseed/validator/admission.go

```diff
@@ -179,6 +179,11 @@ func (v *ManagedSeed) Admit(ctx context.Context, a admission.Attributes, o admis
 	var allErrs field.ErrorList
 	gk := schema.GroupKind{Group: seedmanagement.GroupName, Kind: "ManagedSeed"}
 
+	// Ensure namespace is garden
+	if managedSeed.Namespace != v1beta1constants.GardenNamespace {
+		return apierrors.NewInvalid(gk, managedSeed.Name, append(allErrs, field.Invalid(field.NewPath("metadata", "namespace"), managedSeed.Namespace, "namespace must be garden")))
+	}
```

Comment:
You probably should add the information from your commit message to the comment here so that others don't try to move this back to the static validation.